### PR TITLE
feat(molecule/rating): better implementation for fix custom rating star

### PR DESCRIPTION
**Pull request by juanmaguitar**
Fix custom star. Now the star can be customized in this way...

```js
import {IconStarFilled, IconStarHalfFilled, IconStarOutline} from './Icons'

const customPropsStar = {
  IconStarFilled,
  IconStarHalfFilled,
  IconStarOutline
}

<MoleculeRating
    value={3.5}
    size={MoleculeRatingSizes.LARGE}
    label="25 opiniones"
    {...customPropsStar}
/>
```

- 🌎Demo Online → https://sui-components-fix-custom-star-molecule-rating.now.sh/workbench/molecule/rating/demo

### CODE REVIEW

**Review by jcger — DISMISSED** [review]

File: components/molecule/rating/README.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ import MoleculeRating, {MoleculeRatingSizes} from '@s-ui/react-molecule-rating'
...
 
+### With CUSTOM icons
+
+```js
+import {IconStarFilled, IconStarHalfFilled, IconStarOutline} from './Icons'
+
+const customPropsStar = {
+  IconStarFilled,
+  IconStarHalfFilled,
+  IconStarOutline
```
Comment (rmoralp) [inline components/molecule/rating/README.md]: It should be "IconStarEmpty"

**Review by kikoruiz — DISMISSED** [review]

File: components/molecule/rating/README.md (hunk 1 above)
Comment (rmoralp) [reply components/molecule/rating/README.md]: MoleculeRatingStar has IconStarEmpty, IconStarFilled and IconStarHalfFilled

File: demo/molecule/rating/demo/index.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import {IconStarFilled, IconStarHalfFilled, IconStarOutline} from './Icons'
+
+const BASE_CLASS_DEMO = 'DemoMoleculeRating'
+const CLASS_DEMO_SECTION = `${BASE_CLASS_DEMO}-section`
+const CLASS_DEMO_SECTION_ICONS = `${CLASS_DEMO_SECTION}-icons`
+
+const customPropsStar = {
+  IconStarFilled,
+  IconStarHalfFilled,
+  IconStarOutline
```
Comment (rmoralp) [inline demo/molecule/rating/demo/index.js]: Fix with: {IconStarEmpty: IconStarOutline}

File: components/molecule/rating/README.md (hunk 1 above)
Comment (juanmaguitar) [reply components/molecule/rating/README.md]: You're right. Fixed!

File: demo/molecule/rating/demo/index.js (hunk 2 above)
Comment (juanmaguitar) [reply demo/molecule/rating/demo/index.js]: Fixed!

**Review by jcger — APPROVED** [review]

**Review by rmoralp — APPROVED** [review]

**Pull request merged**